### DISCUSSION TITLE
开发禁用java选项

### DISCUSSION THREAD

**Discussion by yasuo-lang** [Meloong-Git/PCL#5318] [讨论]
下载多个启动器是会自动安装多个java,使PCL自动选择java时会选择很多同版本不同的java,渣机想要在图形显示中选择高性能时要选择很多java很不方便

**Comment by Joker2184** [comment]
（java有区别吗？

你可以直接在版本设置内选择启动java  或者 提 基于游戏版本设置固定java的issue

然后你选成了自定义主页了（（（

**Comment by LinQingYuu** [comment]
>开发禁用java选项

- 你可能指的是 https://github.com/Hex-Dragon/PCL2/discussions/999
如果是，你可以前往上方提到的链接投票，如果不是，可以单开一个 Issue 提出（但大概是被标记为重复）

**Comment by Pigeon0v0** [comment]
为啥会下载多个 Java 啊，已经有的 Java 又不是不会用
